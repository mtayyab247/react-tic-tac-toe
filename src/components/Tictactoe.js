import React from 'react';

function Tictactoe() {
  return (
    <div className="ttt-wrapper">
    
    </div>
  );
}

export default Tictactoe;
