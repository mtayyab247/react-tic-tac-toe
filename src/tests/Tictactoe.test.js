import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Tictactoe from '../components/Tictactoe';

Enzyme.configure({
    adapter: new Adapter()
});

describe("<Tictactoe /> test", () => {
    it("Should render Tictactoe", () => {
        const wrapper = shallow(<Tictactoe />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});


