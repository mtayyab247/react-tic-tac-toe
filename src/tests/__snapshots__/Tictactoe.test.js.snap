// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tictactoe /> test Should render Tictactoe 1`] = `
<div
  className="ttt-wrapper"
>
  <Header
    playerTurn="Player 1"
  />
  <Board
    boardState={
      Array [
        Array [
          1,
          2,
          3,
        ],
        Array [
          4,
          5,
          6,
        ],
        Array [
          7,
          8,
          9,
        ],
      ]
    }
    handleWinner={[Function]}
  />
  <Footer
    handleFinish={[Function]}
    playerTurn="Player 1"
    selectionCount={1}
    winner={false}
  />
</div>
`;
